docs/doc.css

```diff
@@ -0,0 +1,310 @@
+}
+
+.sidebar-main a:hover {
+    background-color: var(--neon-blue);
+    color: var(--main-color);
+}
+
+/* sidebar footer */
+
+.sidebar-footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: var(--neon-blue);
+    padding: 1rem 1rem 1rem 1rem;
+    border-radius: 4rem;
+    font-weight: 700;
+    letter-spacing: 1px;
+    border-top: 2px solid var(--neon-blue);
+    gap: 1rem;
+    margin-top: 0.5rem;
+}
+
+.footer-text {
+    text-align: center;
+    font-size: 1.25rem;
+}
+
+.social-icons {
+    display: flex;
+    justify-content: space-around;
+}
+
+.social-icon {
+    font-size: 1.5rem;
+}
+
+.social-icon:hover {
+    transform: scale(1.4);
+    transition: 0.2s ease-in;
+    cursor: pointer;
+}
+
+/* Main section */
+
+.section {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    overflow-y: scroll;
+    color: var(--black);
+    padding: 2rem;
+    flex-grow: 1;
+}
+
+.section-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding: 1rem;
+    border-bottom: 1px solid #b1acac;
+}
+
+.section-header p {
+    padding-top: 1rem;
+    font-size: 1.2rem;
+}
+
+.section-main {
+    padding: 1rem;
+}
+
+.section-info {
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+}
+
+/* carbon iframe attachment */
+
+.code-img {
+    display: flex;
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+    justify-content: center;
+    width: 100%;
+    height: auto;
+}
+
+/* float button*/
+
+.float-btn {
+    position: fixed;
+    bottom: 1rem;
+    right: 1.2rem;
+    border: none;
+    height: 50px;
+    width: 50px;
+    border-radius: 50%;
+    background-color: var(--main-color);
+    text-decoration: none;
+    box-shadow: 0 0 5px var(--neon-blue);
+}
+
+.float-btn a {
+    color: var(--neon-blue) !important;
+}
+
+.float-btn i {
+    text-shadow: 0 0 4px var(--neon-blue);
+}
+
+.logo h3 {
+    text-shadow: 0 0 5px var(--neon-blue);
+    letter-spacing: 2px;
+}
+
+
+/* scroll bar */
+
+/* Width */
+::-webkit-scrollbar {
+    background-color: transparent;
+    width: 0.75rem;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+    background-color: transparent;
+    border-radius: 1rem;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+    background-color: #01c7ef;
+    border-radius: 10px;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+    background: #01c7efbb;
+}
+
+
+
+
+
+/* Media Queries*/
+
+
+@media screen and (max-width: 600px) {
+
+
+    .sidebar-wrapper {
+        display: none;
+    }
+
+    .section {
+        width: 100%;
+    }
+
+    .section-header,
+    .section-main {
+        padding: 1.5rem 1rem;
+    }
+
+}
+
+@media screen and (max-width: 600px) {
+
+
+    .sidebar-wrapper {
+        display: none;
+    }
+
+    .section {
+        width: 100%;
+        padding: 1rem;
+    }
+
+    .section-header,
+    .section-main {
+        padding: 0.75rem 0.5rem;
+    }
+
+}
```

Comment:
both media-queries have same width of 600px

Reply:
just did that Nikhil ! Thankyou for pointing it out